rpc/block_svc_routines.c

```diff
@@ -3451,14 +3478,19 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
     goto exist;
   }
 
+  if (!resultCaps[GB_CREATE_LOAD_BALANCE_CAP]) {
+    blockGetPrioPath(glfs, blk->volume, list, cobj.prio_path, sizeof(cobj.prio_path));
+  }
+
   uuid_generate(uuid);
   uuid_unparse(uuid, gbid);
 
   GB_METAUPDATE_OR_GOTO(lock, glfs, blk->block_name, blk->volume,
                         errCode, errMsg, exist,
                         "VOLUME: %s\nGBID: %s\n"
-                        "HA: %d\nENTRYCREATE: INPROGRESS\n",
-                        blk->volume, gbid, blk->mpath);
+                        "HA: %d\nENTRYCREATE: INPROGRESS\nPRIOPATH: %s\n",
+                        blk->volume, gbid, blk->mpath,
+                        cobj.prio_path[0] ? cobj.prio_path : "");
```

Comment:
PRIOPATH: line itself shouldn't come when prio_path doesn't exist for backward compatibility right?

Reply:
May be, you are right.